### PR TITLE
Feature/add eslint

### PR DESCRIPTION
**Pull request by rahulrana95**
Fixes #{ISSUE NUMBER}

### Description
I have added the support for eslint for js files.
To run eslint just run the `npm run lint`
_A few sentences describing the change_

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Non Functional Requirement
- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] I have performed a self-review of my own code
- [ ] All new and existing tests passed.
- [X] Documentation

### CODE REVIEW

**Comment by rahulrana95** [comment]
@aashutoshrathi 
I will add the documentation tomorrow regarding rules.

**Comment by rahulrana95** [comment]
@aashutoshrathi  please review this

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,48 @@
+0 info it worked if it ends with ok
```
Comment (singhpratyush) [inline npm-debug.log]: We don't need this file. Please add it to gitignore.

File: package.json [hunk 2]
```diff
@@ -6,10 +6,13 @@
   "version": "0.1.0",
   "description": "Portal to get an invite to IIITV Github organization",
   "scripts": {
-    "start": "node server.js"
+    "start": "node server.js",
+    "lint": "./node_modules/.bin/eslint ./public/*.js *.js",
```
Comment (singhpratyush) [inline package.json]: We can go with simply using `eslint ./public/*.js *.js` here + in the next line.

File: .coafile [hunk 3]
```diff
@@ -0,0 +1,4 @@
+[cli]
```
Comment (singhpratyush) [inline .coafile]: Why do we need a coala config here?

**Comment by aashutoshrathi** [comment]
Any updates on this @rahulrana95

**Comment by aashutoshrathi** [comment]
Done in #64

**Pull request closed without merging**